static/js/lightbox.js

```diff
@@ -210,14 +210,23 @@ function display_image(payload) {
     img.src = payload.source;
     $img_container.html(img).show();
 
-    const filename = payload.url?.split("/").pop();
+    const url = new URL(payload.url, window.location.href);
```

Comment:
I'm not sure I understand the role `window.location.href` has here.

Reply:
If `payload.url` is a relative address, the second argument is [the base](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#base).